scala-client/README.md

```diff
@@ -53,3 +52,27 @@ It's built on top of the java client [apache/incubator-pegasus/java-client](http
     
     c.close
 ```
+
+## Development
+
+### Format the code
+
+Use scala format tool, see https://github.com/scalameta/scalafmt
+```
+sbt scalafmtSbt scalafmt test:scalafmt
+```
+
+### Run tests
+
+Build Java dependency at first, then build and test Scala client.
+```
+cd ${PROJECT_ROOT}/java-client/scripts
+./recompile_thrift.sh
+
+cd ${PROJECT_ROOT}/java-client
+mvn clean package -DskipTests -Dcheckstyle.skip=true
+mvn clean install -DskipTests -Dcheckstyle.skip=true
+
+cd ${PROJECT_ROOT}/scala-client
+sbt test
```

Comment:
Before running test, should steps that build Pegasus and start onebox be added ?

Reply:
Updated.